code/modules/reagents/chemistry_properties/prop_special.dm

```diff
@@ -6,29 +6,16 @@
 /datum/chem_property/special/boosting
 	name = PROPERTY_BOOSTING
 	code = "BST"
-	description = "Boosts the potency of all other properties in this chemical when inside the body."
+	description = "Boosts the potency of all other properties in this chemical when inside the body by up to 4 levels."
 	rarity = PROPERTY_LEGENDARY
 	category = PROPERTY_TYPE_METABOLITE
+	max_level = 4
 
 /datum/chem_property/special/boosting/pre_process(mob/living/M)
+	if(level > max_level)
```

Comment:
dont do this . it will limit admin spawned stuff

Reply:
just make the properties higher in the first place when creating an admin custom chem?